lib/doctest.js

```diff
@@ -18,12 +18,13 @@ var esprima = require ('esprima');
 var _show = require ('sanctuary-show');
 var Z = require ('sanctuary-type-classes');
 
+var common = require ('./common');
 
 function inferType(path) {
```

Comment:
One more empty line after imports, please. :)

Reply:
I remove double newlines compulsively. This is going to keep happening. :P

Reply:
:zap: